@Test
void errorWithThrowable() {
    this.deferredLog.error(this.message, this.throwable);
    this.deferredLog.replayTo(this.log);
    verify(this.log).error(this.message, this.throwable);
}
